Access API/AccessAPI/DTOs/SearchResultDTO.cs

```diff
@@ -1,14 +1,9 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-
-namespace Access_API
+﻿namespace Access_API
 {
     public class SearchResultDTO
     {
-        public string title;
-        public int id;
-        public double score;
+        public string Title;
```

Comment:
Check warning siger "Non-nullable field 'Title' must contain a non-null value when exiting constructor. Consider declaring the field as nullable."